packages/manager/modules/license/src/ducPrice/price.filter.js

```diff
@@ -0,0 +1,93 @@
+import angular from 'angular';
```

Comment:
Is this `ducPrice` filter used elsewhere ? 
Is this a duplicate or do you move this file in `license` module ?

Reply:
@cbourgois  It's a duplicate, because i think it's still used elsewhere in dedicated